src/components/AddItemForm/AddItemForm.js

```diff
@@ -47,11 +103,13 @@ const AddItemForm = ({ listId }) => {
         type="text"
         id="itemName"
         name="itemName"
+        aria-describedby="itemErrorMessage"
         value={formValues.itemName}
         onChange={handleChange}
         maxLength="100"
         required
       />
+      <span id="itemErrorMessage">{errorMessage}</span>
```

Comment:
In addition to aria-invalid on the field, it would be a good idea to also add an `aria-live="assertive"` live region somewhere on this page (or role="alert" attribute, but that was buggy for me in JAWS). A live region will immediately read the text within when it changes, so is useful for form error messages because then they are instantly read when the error occurs. Without this, the user needs to click on the text field to understand that there is an error. 

A common pattern is to use a live region to summarize all form errors (may or may not be displayed as visible to sighted users), then also have hints/descriptions for each field as you have now. In this case, since we only have one field that can have an error, I wonder if we can't just add the `aria-live="assertive"` attribute here on this span? It worked for me in testing anyway!

Reply:
Actually, I'm updating my suggestion after a bunch of testing with focus() and different aria attributes. If you take the suggestion to force focus on the field after the form is submitted with an error, the describedby text will be read automatically and function basically like an alert. So because we only have one field that can have an error right now, we don't need a live region for a "big picture" summary, and setting it on the existing describedby message will just make the screen reader read things twice.